### PR TITLE
Amended ‘home’ breadcrumbs to point to old url.

### PR DESCRIPTION
**Pull request by roncitrus**
Clicking the home link in the breadcrumbs e.g. "Home > Administration > Salutations > Create" should take the user to the url https://chaps.{environment}.net.tp.dsd.io/Chaps_deploy/. where {environment} is either dev, staging or production respectively.

### CODE REVIEW

File: ChapsDotNET/Areas/Admin/Views/Alerts/Create.cshtml [hunk 1]
```diff
@@ -14,9 +15,8 @@
     <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
     <script type="text/javascript" src="@Url.Content("~/lib/jquery-ui-timepicker-addon.js")"></script>
     <script type="text/javascript" src="@Url.Content("~/lib/jHtmlArea-0.8.min.js")" charset="utf-8"></script>
-     
-    <script type="text/javascript">
-        console.log($);
+
+    <script type="text/javascript">console.log($);
```
Comment (wilson1000-MoJ) [inline ChapsDotNET/Areas/Admin/Views/Alerts/Create.cshtml:19]: Hi @roncitrus, I've noticed `console.log` output here - is this intended?

File: ChapsDotNET/Areas/Admin/Views/PublicHolidays/Index.cshtml [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+@model List<ChapsDotNET.Business.Models.PublicHolidayModel>
 
 @{
     ViewData["Title"] = "Public holidays administration";
 }
 
 <div class="breadcrumbs">
     <h6>
-        @Html.ActionLink("Home", "Index", "Home", new { area = "" }, null)
+        <a href="@UrlHelpers.GetTpUrl(Context)">Home</a>
```
Comment (wilson1000-MoJ) [inline ChapsDotNET/Areas/Admin/Views/PublicHolidays/Index.cshtml:10]: I'm starting to wonder why this hasn't been put in a module 😄

File: ChapsDotNET/Common/Helpers/Helpers.cs [hunk 3]
```diff
@@ -30,6 +31,29 @@ public static IHtmlContent Timeago(this IHtmlHelper helper, DateTime? dateTime)
...
+            }
+            else if (currentEnvironment.Contains("dev") || currentEnvironment.Contains("localhost"))
+            {
+                environment = "dev.";
+            }
+            else
+            {
+                environment = "production.";
+            }
+
```
Comment (wilson1000-MoJ) [inline ChapsDotNET/Common/Helpers/Helpers.cs:53]: Hey mate, I have something on multiple if/else if/else statements - something to reduce code.
If you end with an else, you can always define this variable first, reduce logic on the page and reduce visual complexity. Like so:

```csharp
environment = "production.";
if (currentEnvironment.Contains("staging"))
{
    environment = "staging.";
}
else if (currentEnvironment.Contains("dev") || currentEnvironment.Contains("localhost"))
{
    environment = "dev.";
}
```

**Review by wilson1000-MoJ — APPROVED** [review]

File: ChapsDotNET/Areas/Admin/Views/Alerts/Create.cshtml (hunk 1 above)
Comment (roncitrus) [reply ChapsDotNET/Areas/Admin/Views/Alerts/Create.cshtml:19]: Hi @wilson1000-MoJ The console.log is the bit being removed.

Comment (wilson1000-MoJ) [reply ChapsDotNET/Areas/Admin/Views/Alerts/Create.cshtml:19]: Additionally, the output is being duplicated, assuming that `$` and `jQuery` are both referencing the same object. Was this a test to see if either was populated? If so, jQuery should ideally (if I'm not mistaken) be wrapped in a ready object:

```js
(function($){
    // good for compatibility; other JS libraries can use the dollar
    console.log($);
})(jQuery);
```

File: ChapsDotNET/Common/Helpers/Helpers.cs (hunk 3 above)
Comment (roncitrus) [reply ChapsDotNET/Common/Helpers/Helpers.cs:53]: Yeah, that's a good idea. I actually just lifted that bit of code from the old Chaps, missed an opportunity for a bit of refactoring there.

**Pull request merged**